xs2a-adapter-aspsp-registry/src/main/java/de/adorsys/xs2a/adapter/registry/AspspCsvServiceImpl.java

```diff
@@ -61,7 +61,7 @@ private String toCsvString(AspspCsvRecord aspsp) {
         CsvMapper mapper = new CsvMapper();
         CsvSchema schema = mapper.schemaFor(AspspCsvRecord.class).withoutQuoteChar();
 
-        if (aspsp.getAspspName().contains(",")) {
+        if (aspsp.getAspspName().contains(",") && aspsp.getAspspName() != null) {
```

Comment:
null pointer exception will be thrown when ASPSP name is null

Reply:
done